exercises/strain/strain_test.exs

```diff
@@ -0,0 +1,95 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("strain.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure trace: true, exclude: :pending
+
+defmodule StrainTest do
+  use ExUnit.Case
+
+  defp is_odd(n), do: rem(n, 2) == 1
```

Comment:
Since these are functions that return a bool, how about ending them with `?` to follow the convention in the Standard Library?